server/Games/DrawTheWord.js

```diff
@@ -90,10 +90,18 @@ class DrawTheWord extends Game {
       }
     }
   }
-  generateWords() {
-    //Place wword creation stuff here.
-    return ["camping", "knight", "skyline"];
+    generateWords(){
```

Comment:
Might need to remove this, as it seems the word selection / generation exists in both the front and back end. Might as well make it an all front end activity.